build.gradle

```diff
@@ -33,7 +33,7 @@ buildscript {
     dependencies {
         classpath "androidx.navigation:navigation-safe-args-gradle-plugin:$navigationVersion"
 //        classpath "com.google.dagger:hilt-android-gradle-plugin:$hiltVersion"
```

Comment:
Can you please cleanup this while you're here?

Reply:
Done.